CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.15)
+project(libasteroid)
+find_package(Qt5 COMPONENTS Bluetooth REQUIRED)
+if (MSVC)
```

Comment:
It does not really make sense for us to use MSVC does it ?

Reply:
I don't know what you mean.  It might be convenient to have libasteroid as a base for a Windows-based sync.  Is there some reason it shouldn't be included?

Reply:
I'd put Windows support at least in it's own commit, but possible in it's own PR even.

Reply:
Well, there’s no point keeping untested and unmaintained code.
If someone someday actually has a need for this, they can add support then. But I don’t see it coming since there is already winsteroid https://github.com/AsteroidOS/WinSteroid

https://en.m.wikipedia.org/wiki/You_aren%27t_gonna_need_it

Reply:
Oh I didn't know that existed. Last commit in 2018 though, has anyone verified that it still works?

Agreed though, if it's not going to be used then it shouldn't be there.